src/kactl/geometry/insidePolygon.h

```diff
@@ -1,4 +1,5 @@
 /**
+ * Name: Inside polygon (general)
```

Comment:
I'd prefer "Inside general polygon" (and similarly for pseudo-convex), but this is just pedantry